provisionerd/runner/runner.go

```diff
@@ -227,12 +242,18 @@ func (r *Runner) update(ctx context.Context, u *proto.UpdateJobRequest) (*proto.
 }
 
 // doCleanFinish wraps a call to do() with cleaning up the job and setting the terminal messages
-func (r *Runner) doCleanFinish() {
-	// push the fail/succeed write onto the defer stack before the cleanup, so that cleanup happens
-	// before this.
-	var failedJob *proto.FailedJob
-	var completedJob *proto.CompletedJob
+func (r *Runner) doCleanFinish(ctx context.Context) {
```

Comment:
I didn't trace this because it's really only calling 3 traced functions in a row without any logic of its own, might just be best to add it for consistency though or in case any logic gets added here!